src/steps/clusters/index.ts

```diff
@@ -60,6 +62,40 @@ export async function buildClusterUserRelationships({
   );
 }
 
+export async function buildClusterIsAwsInstanceRelationships({
+  instance,
+  jobState,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  // "cloud" is found in the host when Databricks for AWS is used
+  if (!instance.config.databricksHost.includes('cloud')) {
+    return;
+  }
+
+  await jobState.iterateEntities(
+    { _type: Entities.CLUSTER._type },
+    async (clusterEntity) => {
+      await jobState.addRelationship(
+        createMappedRelationship({
+          source: clusterEntity,
+          _class: RelationshipClass.IS,
+          _type: Relationships.CLUSTER_IS_AWS_INSTANCE._type,
+          _mapping: {
+            sourceEntityKey: clusterEntity._key,
+            relationshipDirection: RelationshipDirection.FORWARD,
+            targetFilterKeys: [['tag.ClusterId', '_type', '_class']],
+            targetEntity: {
+              _class: Entities.AWS_INSTANCE._class,
+              _type: Entities.AWS_INSTANCE._type,
+              'tag.ClusterId': clusterEntity.id,
```

Comment:
nit: not a big deal, but I personally like to use raw data (e.g. `cluster.cluster_id`) rather than entity properties (e.g. `clusterEntity.id`) when iterating downstream data, because the typing is more reliable. 

```ts
async (clusterEntity) => {
  const cluster = getRawData<DatabricksCluster>(clusterEntity);
  ...
    'tag.ClusterId': cluster.cluster_id,
    ....
```

https://github.com/JupiterOne/sdk/blob/main/docs/integrations/step-patterns.md#build-child-relationships-step-pattern

Reply:
I see, I was seeing this being used a lot in Azure! Will refactor this :+1: